src/dialog.js

```diff
@@ -66,6 +66,9 @@ window.jQuery = window.jQuery || window.shoestring;
   // default to tracking history with the dialog
   Dialog.history = true;
 
+	// This property is global across dialogs - it determines whether the hash is get/set at all
+	Dialog.useHash = true;
```

Comment:
Does it matter that this will apply to all dialogs on the page? Are there situations where we want it to be "per dialog", could be a quick `this.useHash = this.$el.attr("data-use-hash") || Dialog.useHash` in the constructor.

Reply:
Also, how does this worth with `Dialog.history`? Does `useHash == false` imply that `nohistory == true`?

Reply:
Final note, we should probably avoid binding to `hashchange` when this is false in the jQuery plugin.